src/UndockedRegFreeWinRT/UndockedRegFreeWinRT/typeresolution.cpp

```diff
@@ -0,0 +1,653 @@
+#pragma once
+
+#include <RoMetadataApi.h>
+#include <hstring.h>
+#include <atlbase.h>
```

Comment:
ATL? Is there really a need to take a dependency on that? WRL and WIL provide much more modern tools for doing what you want.

Reply:
I think removed it and I just forgot to remove this header.